validation/start-block_test.go

```diff
@@ -0,0 +1,79 @@
+package validation
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation/internal/bindings"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func testStartBlock(t *testing.T, chain *ChainConfig) {
+	rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+	require.NotEmpty(t, rpcEndpoint)
+
+	client, err := ethclient.Dial(rpcEndpoint)
+	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+	systemConfigAddress := Addresses[chain.ChainID].SystemConfigProxy
+	require.NotZero(t, systemConfigAddress)
+
+	offChainParam := chain.Genesis.L1.Number
+	onChainParam, err := getStartBlockWithRetries(context.Background(), common.Address(systemConfigAddress), client)
```

Comment:
would be nice to use a timeout here

Reply:
Added one here https://github.com/ethereum-optimism/superchain-registry/pull/398/commits/9af8df9638afc1a86a6452329ca867fe99dbdf10